README.md

```diff
@@ -4,59 +4,65 @@ Overview
 [![C/C++ CI](https://github.com/KatanaGraph/katana/actions/workflows/cpp.yaml/badge.svg?branch=master)](https://github.com/KatanaGraph/katana/actions/workflows/cpp.yaml?query=branch%3Amaster)
 [![Python CI](https://github.com/KatanaGraph/katana/actions/workflows/python.yaml/badge.svg?branch=master)](https://github.com/KatanaGraph/katana/actions/workflows/python.yaml?query=branch%3Amaster)
 
-Galois is a C++ library designed to ease parallel programming, especially for
-applications with irregular parallelism (e.g., irregular amount of work in parallel
-sections, irregular memory accesses and branching patterns). It implements
-an implicitly parallel programming model, where the programmer replaces serial loop
-constructs (e.g. for and while) and serial data structures in their algorithms with parallel loop
-constructs and concurrent data structures provided by Galois to express their algorithms.
-Galois is designed so that the programmer does not have to deal with low-level parallel programming constructs such as
-threads, locks, barriers, condition variables, etc.
+Katana is a library and set of applications to work with large graphs efficiently.
 
 Highlights include:
-- Parallel *for_each* loop that handles dependencies between iterations, as well as
-  dynamic work creation, and a *do_all* loop for simple parallelism. Both provide load balancing and excellent
+
+- Parallel graph analytics and machine learning algorithms
+- Multithreaded execution with load balancing and excellent
   scalability on multi-socket systems
-- A concurrent graph library designed for graph analytics algorithms as well as
-  other domains such as irregular meshes.
+- Interfaces to build your own algorithms
 - Scalable concurrent containers such as bag, vector, list, etc.
 
-Galois is released under the BSD-3-Clause license.
+Katana is released under the BSD-3-Clause license.
 
+Installing
+==========
 
-Building Galois
-===============
+The easiest way to get Katana is to install the Conda packages. See the [Conda
+User
+Guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html)
+for details on how to install Conda. 
 
-The master branch will be regularly updated, so you may try out the latest
-development code by checking out master branch:
+```bash
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh
+```
 
-```Shell
-git clone https://github.com/KatanaGraph/katana.git
+Then, install Katana
+
+```bash
+conda install -c katanagraph katana katana-python
```

Comment:
This will not currently work because we do not have any "full" releases. The only packages in conda are in `katanagraph/label/dev`. So we should include getting the development builds here.

Reply:
Updated to the following:

=======================

Then, install Katana

```bash
conda install -c katanagraph/label/dev katana katana-python
```

Currently, only development versions are available through Conda. These builds
reflect the current state of the `master` branch. Releases of stable versions
will be available in the future.